README.template.md

```diff
@@ -207,6 +207,16 @@ Load it using your preferred Emacs package manager, e.g., for Doom Emacs:
 
 ```
 
+## Github's Rate Limit
+
+Github's API has a 60 hit/hour rate-limit. The workaround for this is creating a
+[personal access
+token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+and setting two env vars:
+
+- `BABASHKA_NEIL_DEV_GITHUB_USER`
```

Comment:
Should we just change this to `NEIL_GITHUB_USER` and `NEIL_GITHUB_TOKEN` so it's not dev specific anymore and everyone can use it?

Reply:
@borkdude These are nicer, good thought! I updated the docs and code to read the new env vars. The old ones are still supported for now, but we could drop that support now or in the future - let me know if you feel strongly either way, or if there are any other details you want to address.